contract_manager/scripts/deploy_evm_entropy_contracts.ts

```diff
@@ -5,17 +5,16 @@ import { DefaultStore } from "../src/store";
 import {
   DeploymentType,
   EvmEntropyContract,
-  EvmPriceFeedContract,
   getDefaultDeploymentConfig,
   PrivateKey,
   toDeploymentType,
   toPrivateKey,
-  EvmWormholeContract,
 } from "../src";
 import {
   COMMON_DEPLOY_OPTIONS,
   deployIfNotCached,
   getWeb3Contract,
+  getOrDeployWormholeContract,
 } from "./common";
 import Web3 from "web3";
 
```

Comment:
we have a `DeploymentConfig` type both in this script and the pricefeed script. let's refactor this and extend them from the base `DeployConfig` we have in `common.ts`